_posts/2023-05-23-GUIs-in-PyQt5.md

```diff
@@ -0,0 +1,263 @@
+# Ramin Honary
+
+  - Worked at Cross Compass since 2018
+  - Platform Team development, full stack
+  - Worked primarily on Manufacia
+  - Currently on working on ML-Platform
```

Comment:
```suggestion
```
See here and add formatted header.
https://github.com/xc-jp/blog-posts#building

This is a blog text for external people and self-introduction is not really necessary.